tools/msp_merge/mergeMSP.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+from itertools import chain
+from typing import List
+
+from matchms import Spectrum
+from matchms.exporting import save_as_msp
+from matchms.importing import load_from_msp
+
+
+def read_spectra(filenames: str) -> List[Spectrum]:
+    """Read spectra from files.
+
+    Args:
+        filenames (str): Paths to MSP files from which to load each spectrum.
+
+    Returns:
+        List[Spectrum]: Spectra stored in the file.
+    """    
+    spectra = list(chain(*[load_from_msp(file) for file in filenames]))
+    return spectra
+
+
+def write_spectra(filenames, outfilename):    
+    """Generates MSP file with the merged spectra.
+
+    Args:
+        filenames   (str): Paths to MSP files from which to load each spectrum.
+        outfilename (str): Path to MSP file with merged spectra.
+    """    
+    spectra = read_spectra(filenames)
+    save_as_msp(spectra, outfilename)
+
+
+def merge_spectra(filenames, outfilename):
+    """Save MSP file containing merged spectra.
+
+    Args:
+        filenames   (str): Paths to MSP files from which to load each spectrum.
+        outfilename (str): Path to MSP file with merged spectra.
+    """    
+    return write_spectra(filenames, outfilename)
```

Comment:
I think this one function which only forwards to another function at the moment can be removed and the `read_spectra(...)` and `save_as_msp(...)` calls can be in main directly.

Reply:
Changes applied ([4175a92](https://github.com/wverastegui/galaxytools/commit/4175a928f60307f3718ef64807d12135e070dcf4))